package.json

```diff
@@ -50,10 +50,9 @@
     "cli-unit": "yarn unit-cli",
     "viewer-unit": "yarn unit-viewer",
     "watch": "yarn unit-core --watch",
-    "unit:cicoverage": "yarn c8 --all yarn unit:ci && yarn c8 report --reporter text-lcov > unit-coverage.lcov",
+    "unit:cicoverage": "yarn c8 --all yarn unit:ci",
     "coverage": "yarn unit:cicoverage && c8 report --reporter html",
-    "smoke:cicoverage": "yarn c8 yarn smoke -j=1 --retries=2 && yarn c8 report --reporter text-lcov > smoke-coverage.lcov",
-    "coverage:smoke": "yarn smoke:cicoverage && c8 report --reporter html",
+    "coverage:smoke": "yarn c8 yarn smoke -j=1 && c8 report --reporter html",
```

Comment:
nit: Any reason this can't be `smoke:cicoverage` to match `unit:cicoverage`?

Reply:
> nit: Any reason this can't be `smoke:cicoverage` to match `unit:cicoverage`?

`coverage:smoke` is to match `coverage`...it's meant for if you want to run coverage locally, so it generates an html version of the coverage report for human consumption. I assume it's called `coverage` and not `coverage:unit` because there was only the one kind of coverage back when it was named 🤷